Braintree.xcodeproj/project.pbxproj

```diff
@@ -14,9 +14,13 @@
 		045241352EBEBAEA00C25EA6 /* BraintreeTestShared.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = A903E1A624F9D34000C314E1 /* BraintreeTestShared.framework */; platformFilter = ios; };
 		045241652EBED39400C25EA6 /* BraintreeCore.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 570B93AC285397520041BAFE /* BraintreeCore.framework */; platformFilter = ios; };
 		0452416B2EC4F80F00C25EA6 /* PrivacyInfo.xcprivacy in Resources */ = {isa = PBXBuildFile; fileRef = 045241692EC4F80F00C25EA6 /* PrivacyInfo.xcprivacy */; };
-		0452416C2EC4F80F00C25EA6 /* BTPaymentButtons.swift in Sources */ = {isa = PBXBuildFile; fileRef = 045241682EC4F80F00C25EA6 /* BTPaymentButtons.swift */; };
+		0452416C2EC4F80F00C25EA6 /* BTPayPalButtons.swift in Sources */ = {isa = PBXBuildFile; fileRef = 045241682EC4F80F00C25EA6 /* BTPayPalButtons.swift */; };
 		045241762EC7F14D00C25EA6 /* BraintreePayPal.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 2D941D381B59C76A0016EFB4 /* BraintreePayPal.framework */; platformFilter = ios; };
 		045241772EC7F15400C25EA6 /* BraintreeVenmo.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = A77AA29C1B618C7700217B73 /* BraintreeVenmo.framework */; platformFilter = ios; };
+		046F006C2ECBF46800F08512 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 046F006B2ECBF46800F08512 /* Assets.xcassets */; };
+		046F006E2ECBF46800F08512 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 046F006B2ECBF46800F08512 /* Assets.xcassets */; };
+		046F00742ECCEFC800F08512 /* BTPayPalButtonColor.swift in Sources */ = {isa = PBXBuildFile; fileRef = 046F00732ECCEFC300F08512 /* BTPayPalButtonColor.swift */; };
+		046F00762ECE4E1400F08512 /* BTUIComponents+Extensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 046F00752ECE4E0B00F08512 /* BTUIComponents+Extensions.swift */; };
```

Comment:
It looks like some of these may be duplicate files from the feature branch. Are you able to double check?